presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -4704,11 +4704,8 @@ public void testAccessControl()
         assertAccessAllowed("SHOW STATS FOR lineitem", privilege("orders", SELECT_COLUMN));
         assertAccessAllowed("SHOW STATS FOR (SELECT * FROM lineitem)");
         assertAccessAllowed("SHOW STATS FOR (SELECT * FROM lineitem)", privilege("orders", SELECT_COLUMN));
-        assertAccessDenied("SHOW STATS FOR (SELECT * FROM lineitem)",
-                "Cannot select from columns \\[returnflag, linenumber, orderkey, quantity, shipmode, discount, tax, suppkey, partkey, shipinstruct, linestatus, extendedprice, comment, receiptdate, commitdate, shipdate] in table or view .*.lineitem.*",
-                privilege("lineitem", SELECT_COLUMN));
-        assertAccessDenied("SHOW STATS FOR (SELECT nationkey FROM nation)", "Cannot select from columns \\[nationkey\\] in table or view .*.nation.*", privilege("nation", SELECT_COLUMN));
-        assertAccessDenied("SHOW STATS FOR (SELECT * FROM nation)", "Cannot select from columns \\[nationkey, regionkey, name, comment\\] in table or view .*.nation.*", privilege("nation", SELECT_COLUMN));
+        assertAccessDenied("SHOW STATS FOR (SELECT * FROM nation)", "Cannot show stats for columns \\[nationkey, regionkey, name, comment\\] in table or view .*.nation", privilege("nation", SELECT_COLUMN));
```

Comment:
undo formatting, it is hard to follow what changed actually.

Reply:
it's not formatting change. Message of the exception did change.

Reply:
Yes, but previously this assertion was written in 3 lines, now it is in a single line. I prefer 3 lines per each assertion. Less horizontal scrolling for me ;)